doc.go

```diff
@@ -1,13 +1,43 @@
 /*
 Package braintree is a client library for Braintree.
 
+Initializing
+
+Initialize it with API Keys:
+
+	braintree.New(Sandbox, merchantId, publicKey, privateKey)
+
+Initialize it with an Access Token:
+
+	braintree.NewWithAccessToken(accessToken)
```

Comment:
I added a comment to the NewWithAccessToken function that when that is used webhooks are unsupported.